lib/chef/http.rb

```diff
@@ -248,8 +248,6 @@ def http_client(base_url = nil)
       end
     end
 
-    private
```

Comment:
Is this because rubocop dislikes the `public` we had down below? Probably better to move the one method that affected and leave the rest `private` ?